src/frontends/tests/frontend/shared/src/basic_api.cpp

```diff
@@ -16,7 +16,6 @@ std::string FrontEndBasicTest::getTestCaseName(const testing::TestParamInfo<Basi
 }
 
 void FrontEndBasicTest::SetUp() {
-    FrontEndTestUtils::setupTestEnv();
```

Comment:
Shouldn't you remove `setupTestEnv` function at all?

Reply:
Yes, I'll do it later when will remove all mentions of OV_FRONTEND_PATH